src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/books';
+import categoriersReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  books: booksReducer,
+  categories: categoriersReducer,
+});
+
+const store = configureStore({ reducer: rootReducer });
```

Comment:
- [OPTIONAL] 🤔 What If you Import `applyMiddleware`  to use some handy Redux middleware, like a redux logger.  it will help you a lot in the development process. 👍🏻 To install it, you need to run: 

  ```  npm i --save redux-logger  ```